### PR TITLE
Feature/react intl the rest

### PR DESCRIPTION
**Pull request by UlyanaHashchuk**
## Info
Added Intl messages for: 
  - `Sign in` page.
  - `Logout` button.
  - `Coming soon` section which is in `Transactions` and `Analytics` pages.
  - The rest minor sentences in Vanilla Admin

### CODE REVIEW

**Comment by UlyanaDurcanska** [comment]
Changes started from commit 'added react-intl messages for Setup section'. I do not have conflicts, created new branch, but somehow still see previous commits.

File: pages/_app.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const App = ({
       </Head>
       <ThemeProvider theme={theme}>
         <GlobalStyle theme={theme} />
-        <IntlProvider locale='en'>
+        <IntlProvider locale="en">
```
Comment (norbertdurcansk) [inline pages/_app.tsx:28]:
```suggestion
        <IntlProvider locale='en'>
```

Comment (norbertdurcansk) [inline pages/_app.tsx]:
```suggestion
        <IntlProvider locale='en'>
```

**Review by norbertdurcansk — APPROVED** [review]
Good job 💯

**Pull request merged**